tests/test_send_recv_two_workers.py

```diff
@@ -43,7 +43,10 @@ def client(env, port, func):
     # assert deserialized msg is cdf
     # send receipt
 
+    ucp.reset()
     os.environ.update(env)
+    ucp.init()
```

Comment:
Does using `reset()` and `init()` now work? Had [some issues]( https://github.com/rapidsai/ucx-py/issues/290#issuecomment-544996019 ) with this previously.

Reply:
yes, they should be working now.  I think this is only necessary when running all tests together -- pytest, i think does something strange here